bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/v19/BitcoindV19RpcClient.scala

```diff
@@ -36,6 +39,23 @@ class BitcoindV19RpcClient(override val instance: BitcoindInstance)(
     with PsbtRpc
     with V19BlockFilterRpc {
 
+  override def getFiltersBetweenHeights(
+      startHeight: Int,
+      endHeight: Int): Future[Vector[ChainQueryApi.FilterResponse]] = {
+    Future.sequence(
+      startHeight
+        .until(endHeight)
+        .map { height =>
+          for {
+            hash <- getBlockHash(height)
+            filter <- getBlockFilter(hash, FilterType.Basic)
+          } yield FilterResponse(filter.filter, hash, height)
+        }
+        .toVector)
+  }
+
+  override def getFilterCount: Future[Int] = getBlockCount
```

Comment:
I don't think this is right, is there really no `getfiltercount` in any bitcoind version?

Reply:
No, after getting a block it'll calculate the filter immediately so there should never be a point where they are different.